.github/workflows/cross_platform_ci.yml

```diff
@@ -0,0 +1,102 @@
+name: rviz cross-platform RoboStack build
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - "[kmn]*-devel*"
+
+jobs:
+  run_rviz_compilation:
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Build Dependencies
+      uses: mamba-org/provision-with-micromamba@main
+      with:
+        environment-file: .github/ci_cross_platform_env.yml
+        micromamba-version: latest
+
+    - name: Set up rviz Dependencies on Unix
+      if: runner.os == 'Linux' || runner.os == 'macOS'
+      shell: bash -l -eo pipefail {0}
+      run: |
+        micromamba activate test
+        export PATH=$HOME/micromamba-bin:$PATH
+        rosdep init
+        rosdep update
+        rosdep install --from-paths . --ignore-src -r -y
+
+    - name: Build rviz on Unix
+      if: runner.os == 'Linux' || runner.os == 'macOS'
+      shell: bash -l -eo pipefail {0}
+      run: |
+        if [[ `uname -s` == "Linux" ]]; then
+          CDT="-cos6-x86_64"
+          micromamba install -y mesa-libgl-devel$CDT mesa-dri-drivers$CDT \
+                                libselinux$CDT libxdamage$CDT libxxf86vm$CDT \
+                                libxext$CDT libxfixes$CDT -c conda-forge
+        fi
```

Comment:
nitpick: Shouldn't that be part of the previous step (`Set up rviz Dependencies on Unix`)?